spec/controllers/topics_controller_spec.rb

```diff
@@ -882,6 +882,46 @@
         Topic.any_instance.expects(:set_auto_close).with(nil, anything)
         xhr :put, :autoclose, topic_id: @topic.id, auto_close_time: nil, auto_close_based_on_last_post: false, timezone_offset: -240
       end
+
+      it "will close a topic when the time expires" do
+        topic = Fabricate(:topic)
+        Timecop.freeze(Time.now - 20.hours) do
```

Comment:
20.hours.ago. Also, in a rails app that's using TimeWithZone, Time.now is a Bad Idea. Always use Time.zone.now. But in this case, 20.hours.ago is best.